pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1014,7 +1027,7 @@ func (self *awsInstance) getMountDevice(volumeID string, assign bool) (assigned
 		if strings.HasPrefix(name, "/dev/xvd") {
 			name = name[8:]
 		}
-		if len(name) != 1 {
+		if len(name) < 1 {
```

Comment:
The intention was to catch 'dev' still in there.  How about we check for len(name) < 1 || len(name) > 2